src/App.css

```diff
@@ -9,8 +9,21 @@ body {
   text-align: center;
 }
 
-.navBar {
-  /* background-image: url(/purplesky.png); */
+.date-button{
+  text-transform: uppercase;
+  margin-right:1rem; 
+  border: none;
+  background-color:white;
+}
+
+.button-active{
+  background-color:  #737495!important;
+  color: #FFF;
+  text-decoration: none;
+}
+
+
+.navbar {
   font-family: "Cuprum", sans-serif;
```

Comment:
As you can see I change naming structure. EG"navBar" to "nav-bar". This is the right way to name classes and id in CSS.

Reply:
Homework: restructure other names like that.